app/src/main/java/edu/wpi/first/shuffleboard/app/WidgetPaneController.java

```diff
@@ -39,15 +47,19 @@
 import javafx.scene.layout.GridPane;
 import javafx.scene.layout.Region;
 
+// needs refactoring to split out per-widget interaction
+@SuppressWarnings("PMD.GodClass")
```

Comment:
PMD is right, this class is doing too much right now.

Reply:
Yes but I can't address that in this PR.